process/working-group-lifecycle.md

```diff
@@ -8,14 +8,29 @@ However, if no existing effort covers the topic feel free to make use of the exi
 
 Please note that running a WG successfully requires an extended effort. Be prepared to commit at least several hours per week for a year or more.
 
-The WG Life Cycle begins with establishing commitments from other interested contributors then making a proposal to the TAC. If accepted, the WG will begin as `Incubating`. This is a boot strapping phase for the group to adopt OpenSSF community policies and norms. Once those are met the group becomes `Active` and will be fully recognized in OpenSSF. Either upon completing its objectives or no longer making forward progress a project will move to `End-of-Life`. The details for each phase follow.
+The WG Life Cycle begins with establishing commitments from other interested contributors then making a proposal to the TAC. If accepted, the WG will begin as `Samdbox`. This is a boot strapping phase for the group to define and document what their goals are. 
+adopt OpenSSF community policies and norms. Once those are met the group becomes `Active` and will be fully recognized in OpenSSF. Either upon completing its objectives or no longer making forward progress a project will move to `End-of-Life`. The details for each phase follow.
 
-## To become `Incubating`:
+## To become `Sandbox`:
 
 * Proposal of scope for review by TAC
-    * This is to help ensure limited overlap with existing WG
+    * This is to help ensure limited overlap with existing WGs
+* Have at least 3 interested individuals from different organizations supporting the proposal
+* TAC will vote to approve or provide constructive guidance
+
+## Once `Sandbox`:
+
+* The TAC will add the WG to the list of WGs to its README.
+* If the WG has meetings:
```

Comment:
I was confused by this at first, because I was thinking "of course a WG must have meetings!" But I think what we're saying is "If a WG has meetings **at the sandbox stage**"? If so, we might want to make that more explicit.

Reply:
I added your piece to make it clear.